gobblin-config-management/gobblin-config-client/src/test/java/gobblin/config/client/TestConfigClient.java

```diff
@@ -131,10 +131,10 @@ public void setup(){
   private void mockupConfigValues(){
     /**
      * each node will have a common key "generalKey" with value as "generalValue_${node}"
-     * this key will be overwrite 
-     * 
+     * this key will be overwrite
```

Comment:
A lot of format changes, please file a separate PR for format changes in the future, then we do NOT need to "really" review those changes and reduce a lot of noise.